### PR TITLE
PostWorkshopAction: link to survey results

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This PR adds a survey result link (`reports_link`) to `PostWorkshopAction` context.

### CODE REVIEW

File: amy/autoemails/tests/test_postworkshopaction.py [hunk 1]
```diff
@@ -197,6 +197,7 @@ def testContext(self):
                 helpers=[p1, p3],
                 all_emails=['hg@magic.uk', 'hp@magic.uk'],
                 assignee='Regional Coordinator',
+                reports_link='https://workshop-reports.carpentries.org/?key=e18dd84d093be5cd6c6ccaf63d38a8477ca126f4slug=test-event',
```
Comment (fmichonneau) [inline amy/autoemails/tests/test_postworkshopaction.py:200]: missing `&` between the 2 URL parameters

File: config/settings.py [hunk 2]
```diff
@@ -611,3 +611,19 @@
...
+REPORTS_SALT_BACK = env.str('AMY_REPORTS_SALT_BACK', default='')
+if not DEBUG and not (REPORTS_SALT_FRONT and REPORTS_SALT_BACK):
+    raise ImproperlyConfigured(
+        "Report salts are required. See REPORT_SALT_FRONT and REPORT_SALT_BACK"
+        " in settings."
+    )
+
+REPORTS_LINK = env.str(
+    'AMY_REPORTS_LINK',
+    default='https://workshop-reports.carpentries.org/?key={hash}slug={slug}',
```
Comment (fmichonneau) [inline config/settings.py:628]: the value for the environment variable is correct in https://github.com/carpentries/systems/pull/67/files but the default is missing the "&" between key and slug I think.

**Comment by pbanaszkiewicz** [comment]
@fmichonneau great catch! I would scratch my head in future if it stayed that way. 😄

**Comment by fmichonneau** [comment]
thanks!

**Pull request merged**